java-shared-dependencies/unmanaged-dependency-check/src/main/java/com/google/cloud/UnmanagedDependencyCheck.java

```diff
@@ -18,11 +18,13 @@
  * A utility class to check unmanaged dependencies in BOM.
  */
 public class UnmanagedDependencyCheck {
+  // regex of handwritten artifacts
+  private final static String handwrittenArtifact = "(com.google.cloud:google-cloud-.*)|(com.google.api.grpc:(grpc|proto)-google-cloud-.*)";
```

Comment:
"proto-" or "grpc-" is not handwritten artifact. Can you come up with a better name?

Reply:
How about `downstreamArtifact` as they are artifacts of of downstream repositories?